### ISSUE TITLE
Spyder 2 theme is showing icons in menus on Mac

### ISSUE THREAD

**Issue by ShenggaoZhu** [spyder-ide/spyder#2909]
## Description

With Spyder 3 theme, icons are not showing in the menu:

![image](https://cloud.githubusercontent.com/assets/11516238/12171140/4a43d26c-b582-11e5-816b-81a1ab35ed04.png)

Spyder 2 theme is working fine:
![image](https://cloud.githubusercontent.com/assets/11516238/12171169/6a9160f2-b582-11e5-815f-638b2c8d8730.png)

## Version and main components

* Spyder Version: 3.0.0b2 
* Python Version: 2.7.11
* Qt Versions: 5.4.1, PyQt5 5.4.1 on Darwin

## Dependencies
```
pyflakes >=0.5.0   :  1.0.0 (OK)
pep8 >=0.6         :  1.6.2 (OK)
pygments >=1.6     :  2.0.2 (OK)
IPython >=3.0      :  4.0.1 (OK)
zmq >=13.0.0       :  15.1.0 (OK)
qtconsole >=4.0    :  4.1.1 (OK)
pandas >=0.13.1    :  0.17.1 (OK)
numpy >=1.7        :  1.10.1 (OK)
sphinx >=0.6.6     :  1.3.3 (OK)
psutil >=0.3       :  3.3.0 (OK)
rope >=0.9.4       :  0.9.4-1 (OK)
jedi >=0.8.1;<0.9.0:  0.8.1 (OK)
matplotlib >=1.0   :  1.5.0 (OK)
sympy >=0.7.3      :  0.7.6.1 (OK)
pylint >=0.25      :  1.5.2 (OK)

```

**Comment by ccordoba12** [comment]
Thanks for reporting. This is an bug in PyQt5 that i already noticed.

**Comment by ccordoba12** [comment]
The solution is to force Qt to *not* show icons on menus because that's the standard on Mac.

**Comment by ShenggaoZhu** [comment]
Thanks @ccordoba12 for the explanation! Didn't know about that Mac standard before... 
The menu icons in Spyder look nice to me, and I just checked, Eclipse on Mac still shows icons in the menus. But I guess it's a good thing for Spyder to respect the standards of different platforms... Hope it won't be too difficult to implement!

**Comment by ccordoba12** [comment]
This was already fixed in PR #3501.
